Bitbucket.Authentication/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers by using the '*'
 // as shown below: [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
@mminns is starting with v1.0.0 alright with you? I know your original contribution was at v1.1.0 but I assumed that was a copy-paste error. If v1.1.0 is necessary for some reason, I'll be happy to revert this change.